jdaviz/conftest.py

```diff
@@ -156,6 +158,30 @@ def spectrum_collection(spectrum1d):
     return result
 
 
+@pytest.fixture
+def multi_order_spectrum_list(spectrum1d, spectral_orders=10):
+    sc = []
+
+    for i in range(spectral_orders):
+        np.random.seed(42)
```

Comment:
Do you mean to set the seed inside the loop? Won't all the spectra actually have the same values like this?

Reply:
Yikes, thanks!

Reply:
D'oh! Fixed.